core/src/test/scala/com/databricks/labs/remorph/transpilers/SnowflakeToDatabricksTranspilerTest.scala

```diff
@@ -148,6 +148,20 @@ class SnowflakeToDatabricksTranspilerTest extends AnyWordSpec with TranspilerTes
            |  t1;""".stripMargin
     }
 
+    "transpile ARRAY_REMOVE function" ignore {
+      // TODO: enable the test once the parsing issue is fixed
+      // https://github.com/databrickslabs/remorph/issues/978
+      "SELECT ARRAY_REMOVE([1, 2, 3], 1);" transpilesTo
+        "SELECT ARRAY_REMOVE(ARRAY(1, 2, 3), 1);"
+
+      "SELECT ARRAY_REMOVE([2, 3, 4.00::DOUBLE, 4, NULL], 4);" transpilesTo
+        "SELECT ARRAY_REMOVE(ARRAY(2, 3, 4.00::DOUBLE, 4, NULL), 4);"
```

Comment:
This transpiles to:

```sql
ARRAY_REMOVE(ARRAY(2, 3, 4::DOUBLE, 4, NULL), 4)
```

Though poerhaps it should not. Tests added in my PR to fix array literal parsing